Automattic/jetpack
# My Jetpack: make card primary button links when applicable

Fixes #31616 #31619

## Proposed changes:

In the _My Jetpack_ section, activating the primary button of product cards often directs users to a different page. They're effectively links rather than buttons but are always implemented with the `button` tag. Using the proper semantics is important: it allows, for instance, screen reader users to better identify links present on the page.

This PR implements links when applicable.

### Other information:


## Jetpack product discussion


pedMtX-Rg-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Spin up a test site with this branch
* Visit the _My Jetpack_ page
* For each possible action (_Activate_, _Manage_, _Purchase_, _Start for Free_, _Install_), check that the behaviour is unchanged

<img width="500" alt="Screenshot 2023-06-27 at 2 11 11 PM" src="https://github.com/Automattic/jetpack/assets/1620183/e1d70dc7-e834-43a9-9d47-a0f3ef4e577e">
